weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1371,11 +1371,13 @@ def _select_objs_query(
         conditions: Optional[list[str]] = None,
         limit: Optional[int] = None,
         parameters: Optional[Dict[str, Any]] = None,
+        is_latest: bool = False,
```

Comment:
this is the only thing that can't actually go in the heavy filter, because it is really the reason for the subquery